src/Untagged/Castable.purs

```diff
@@ -6,11 +6,12 @@ module Untagged.Castable
 
 import Foreign (Foreign)
 import Literals.Undefined (Undefined)
-import Prim.RowList (class RowToList, Cons, Nil, kind RowList)
+import Prim.RowList (class RowToList, Cons, Nil, RowList)
 import Unsafe.Coerce (unsafeCoerce)
 
 --| A `Castable a b` exists if all values of type `a` have
 --| runtime values that can be interpreted as that of type `b`.
+class Castable :: forall a b. a -> b -> Constraint
```

Comment:
@jvliwanag Do you anticipate the classes in this library being used with kinds other than `Type`? If so then this is fine; if not, then it's better to just fix them to `Type`:

```purs
class Castable :: Type -> Type -> Constraint
```

Reply:
Yes, fixing them to `Type` is best since we're only dealing with those which have values.